GameOfLife/Calculations/Rules.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    class Rules
+    {
+        public int[,] CellCalculation(int[,] GameField)
+        {
+            int[,] tempArr = (int[,])GameField.Clone();
+            int lenght = GameField.GetLength(0);
+            int width = GameField.GetLength(1);
+
+            for (int i = 0; i < lenght; i++)
+            {
+                for (int j = 0; j < width; j++)     // set copied same size array elements to 0
+                {
+                    tempArr[i, j] = 0;
+                }
+            }
+
+
+            for (int i = 0; i < lenght ; i++)
+            {
+                for (int j = 0; j < width ; j++)
+                {
+                    int liveCells = 0; // Livecell neighbour count for each cell
```

Comment:
Also, one thing about comments in code. Typically, you would write them above the code line you are adding comment to:
```
// Livecell neighbour count for each cell
int liveCells = 0;
```

And, think about, what value does the comment bring to the code in this case (And in other places in code). If you removed this comment, would we still understand what is happening here? A good comment would tell you why something is being done, not what, because the what part we can see from code it self (Unless its some crazy algorithm, that will lose meaning to you the moment you look away from it :D).